src/styles/emptyAndError.styl

```diff
@@ -1,14 +1,13 @@
 @require 'cozy-ui'
 
-.pho-empty
+.pho-empty, .pho-error
```

Comment:
I *think* we generally put these selectors on multiple lines, but it might just be personal preferences :-)